parakeet_rocm/integrations/stable_ts.py

```
@@ -0,0 +1,203 @@
+                "transcribe_any yielded 0 segments;"
+                "falling back to postprocess_word_timestamps(...)"
+            )
+            processed = stable_whisper.postprocess_word_timestamps(
+                {"segments": [segment]},
+                audio=str(audio_path),
+                **options,
+            )
+            segments_out = (
+                processed.get("segments", []) if isinstance(processed, dict) else []
+            )
+    except Exception as e:  # pragma: no cover - fallback path
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)